src/main/resources/assets/jei/lang/de_DE.lang

```diff
@@ -1,64 +1,88 @@
 # Tooltips
-jei.tooltip.config=Konfiguration
+jei.tooltip.config=JEI-Konfiguration
+jei.tooltip.show.recipes=Zeige Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
 jei.tooltip.delete.item=Klicke zum Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
+jei.tooltip.transfer=Bewege Gegenstände
+jei.tooltip.recipe.ore.dict=Akzeptiert: %s
+jei.tooltip.item.colors=Farben: %s
+jei.tooltip.shapeless.recipe=Formloses Rezept
+jei.tooltip.cheat.mode=Cheat-Modus
 
 # Error Tooltips
-jei.tooltip.error.recipe.transfer.missing=Fehlende Items
-jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist voll
+jei.tooltip.error.recipe.transfer.missing=Fehlende Gegenstände
```

Comment:
It's not an issue that appears in Vanilla, meaning that it will never be fixed. That's how it works.

Forge, on the other hand, won't fix it because this would require a ground-up rewrite of the font renderer and GUI system.